### ISSUE TITLE
Restore Echo effects and enable true Stereo Audio (maybe restore Dolby Pro Logic II too)

### ISSUE THREAD

**Issue by hyperspeedgx** [nipkownix/re4_tweaks#300]
### Describe the feature you'd like to see added

In the GC and Wii version the game has echo/reverb in some areas inside the Castle, peharps this can be restored?

Also, PC ver. is know for be a mess in audio, there are more than echo/reverb effects missing, game runs at Mono audio quality while BGM seems to be the only thing playing at Stereo, some areas when you are playing the GC or Wii ver. you can hear much more sounds from the enviroment, like water sounds etc, if you do a playthough of GC or Wii with headphones you will notice even more, that's why some people still play the game in the emulator, audio is just at another level, can't say anything more.

btw: Dunno if GC had this but Wii and even the PS2 ver. has support to Dolby Pro Logic II, maybe this can be restored too?

**Comment by albertre4HD** [comment]
It's true, certain effects lack its positionality in this port, but most of them still sound more to the left or to the right according to the sound source location.
Nevertheless, there's something broken/missing about the sound, yes.
The reverb sound is something I added to my "wishlist" a long ago, so, we are aware of that.
I really hope something can be done with that eventually! :D

**Comment by sologuitarplaying** [comment]
Reloading and gunfire sound stronger and richer on GC and Wii as well. Whatever happened to the sound effects over the years of ports downgraded every sound effect in the game.

**Comment by albertre4HD** [comment]
But the sound samples are the same. What's different is how they are treated by the game.

We are lucky this PC port has a good low-frequency sounds resample method. You only need to hear the ganados' voices and you'll clearly notice they sound better than in ps4/Xone/switch later ports, and they sound almost as good as in the GC. (positional audio is another story, I'm just talking about the perceived audio frequency, how clear or muffled they sound)

But as I said, how the sounds are resampled is what makes the difference. They are in all ports (but PS2) the same sound samples, some 32kz, some 22KZ, 11KZ, 8KZ...

Of course, higher frequency sounds will sound good in all ports, the difference is perceived with the lower frequency sounds. A good place to compare it is when Leon reaches the center of the Village and he uses the binoculars (most ganado's voices and certain enemies' sounds like the hayfork)

If the reloading and gunfire sound different, then it's probably because the game lacks some sound processing step --or however it's called-- the Gamecube did.

Maybe it's related to the missing reverb effect and it's all the same problem. Reverb effects add certain richness to the sounds, after all.

**Comment by pas-de-2** [comment]
I've been wondering about this. If you stand here, you'll notice the hay tending sound in village seems to lack stereo, but if you kill either of the two Ganados tending hay with Disable Enemy Targeting enabled, it seems like the problem might be more that they're just too loud?

![20221231154805_1](https://user-images.githubusercontent.com/116680301/210155494-4eb4ae72-ec85-4f26-b49b-a87cb4b4fcae.jpg)

Similar here, if you kill one of these two Ganado tending hay, you can hear their audio is in stereo. It's just super loud for some reason.

![20221231162626_1](https://user-images.githubusercontent.com/116680301/210155902-08071998-c581-4c39-adcc-bb94ea010932.jpg)

The real offender though is the sound of footsteps. Those really do sound like they're in mono. You can hear the Ganado on the second floor of the barn as if he was walking right on top of you. It gives the impression that it's actually you walking on top of wood, which is strange when you're walking on dirt.

**Comment by albertre4HD** [comment]
Yeah.. the game also has some positional sound issues. Sometimes it looks like the enemies are really close because of the volume of their steps, It's a bit messed up. I haven't paid a lot of attention about what are the exact sound issues other than these positional problems, the lack of the gamecube reverb effect in certain rooms, and some "click" sounds when a ganado says something

**Comment by Nitsug72** [comment]
Hi. Apparently this doesn't happen with bluetooth devices (????)

Whenever I use my akg y500 I don't have this issue, but I do with any cable-headphones.
